### PR TITLE
Go1.19, whitesource removal & minor formatting changes

### PR DESCRIPTION
**Pull request by elsesiy**
* d2aa4b6 Replace ioutil with non-deprecated equivalents
* 6a63aec Fix protoc install instruction
* ecfabce Fix import ordering
* 123cd4a Remove whitesource
* 1150092 Update go toolchain & deps for 1.19

### CODE REVIEW

**Comment by elsesiy** [comment]
@sfc-gh-jchacon Please take a look, happy to discuss if there are questions

File: cmd/proxy-server/main.go [hunk 1]
```diff
@@ -25,18 +25,19 @@ import (
 	"os"
 	"strings"
 
+	"github.com/go-logr/logr"
+	"github.com/go-logr/stdr"
```
Comment (sfc-gh-jchacon) [inline cmd/proxy-server/main.go:29]: if we're going to be grouping make the logr one a separate group too

**Review by sfc-gh-jchacon — APPROVED** [review]

File: cmd/proxy-server/main.go (hunk 1 above)
Comment (elsesiy) [reply cmd/proxy-server/main.go:29]: It's a 3rd party dependency so I'm curious to hear why we would separate it into it's own group? I've been strictly following https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e

File: services/util/util.go [hunk 2]
```diff
@@ -153,9 +154,10 @@ func NewLimitedBuffer(max uint) *LimitedBuffer {
 // buffer has reached the max size no more bytes will be added.
 // TODO: Implement remaining bytes.Buffer methods if needed.
 // NOTE: This is not an error condition and instead no more bytes
-//       will be written and normal return will happen so writes
-//       so not fail. Use the Truncated() method
-//       to determine if this has happened.
+//
```
Comment (sfc-gh-jchacon) [inline services/util/util.go:157]: why this newline?

File: services/util/util.go [hunk 3]
```diff
@@ -153,9 +154,10 @@ func NewLimitedBuffer(max uint) *LimitedBuffer {
...
+//	will be written and normal return will happen so writes
+//	so not fail. Use the Truncated() method
```
Comment (sfc-gh-jchacon) [inline services/util/util.go:159]: do not fail.

**Review by sfc-gh-jchacon — APPROVED** [review]

**Pull request merged**